config/src/config/admission_control_config.rs

```diff
@@ -18,8 +18,9 @@ pub struct AdmissionControlConfig {
 impl Default for AdmissionControlConfig {
     fn default() -> AdmissionControlConfig {
         AdmissionControlConfig {
-            address: "0.0.0.0".to_string(),
-            admission_control_service_port: 8000,
+            address: "0.0.0.0:8000"
```

Comment:
I think in some of these cases and unwrap is good enough.

Reply:
Done, thanks!